picard/ui/coverartbox.py

```diff
@@ -255,28 +283,29 @@ def load_remote_image(self, url, mime, data):
         try:
             coverartimage = CoverArtImage(
                 url=url.toString(),
+                types=[u'front'],
                 data=data
             )
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
         if isinstance(self.item, Album):
             album = self.item
-            album.metadata.append_image(coverartimage)
+            album.metadata.set_front_image(coverartimage)
```

Comment:
Sure, when an user drags an image and drops it on the cover art box, I think the natural thing he/she wants to do is to replace the cover art with something different, not add another image to a list of images. Maybe in the context menu (check my cover-art-context-menu branch) I can add a "radio button" where user can select between "Append images on drop" or "Replace images on drop". Would that be better? Still, I think the default should be "replace" since it's more intuitive.